src/components/UI/Button/style.css

```diff
@@ -0,0 +1,87 @@
+/* @import "../../../styles/variables"; */
+
+.button {
+  cursor: pointer;
+  border: none;
+  padding: 1rem 2rem;
+  user-select: none;
+  text-decoration: none;
+  color: var(--white);
+  border: 1px solid var(--black);
+  background: var(--black);
+  min-width: 10rem;
+  margin: 8px;
+}
+
+.pill {
+  border-radius: 50px;
+}
+
+.rounded {
+  border-radius: 5px;
+}
+
+.buttonOutline {
```

Comment:
css is using => `.button-outline`. css-module shoul transform that to camelCase automatically

Reply:
ok noted, removing this stylesheet later on anyway. all the styles would be in the JS file.

Reply:
still there ?